src/models/forum.rs

```diff
@@ -0,0 +1,10 @@
+//! The Forum struct
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+pub struct Forum {
+    pub id: &'static str,
+    pub name: &'static str,
+    pub description: &'static str,
```

Comment:
These shouldn't be `&'static str`; this means that the string will be available for the entire runtime of the program. Once we start pulling from the database, this will not be the case.

The `id` should probably be a newtype struct over a `Box<str>`, since this won't change. The `name` and `description`, on the other hand, would be mutable at some point; they should be `String`s.

Reply:
Thanks for the suggestion! I've adjusted these types as you suggest